telegram_send.py

```diff
@@ -209,6 +212,16 @@ def send_message(message):
             for i in images:
                 bot.send_photo(chat_id=chat_id, photo=i)
 
+    if locations:
+        it = iter(locations)
+        for loc in it:
+            if ',' in loc:
+                lat, lon = loc.split(',')
+            else:
+                lat = loc
+                lon = next(it)
+            bot.send_location(chat_id=chat_id, latitude=float(lat), longitude=(lon))
```

Comment:
`longitude=(lon)`, should that be `longitude=float(lon)`?

Reply:
yep, sorry about that